src/cards/ResourceCard/ResourceCard.js

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Typography, Button, Card, CardHeader, CardContent, CardActions, Tooltip, Fade } from '@material-ui/core';
+import { DefaultAvatar } from '../../misc';
+
+import useStyles from './style';
+
+export const ResourceCard = (props) => {
+  const classes = useStyles();
+  const { id, name, description, action } = props;
+
+  const getActionButton = () => {
```

Comment:
Why do we need to create a function for CardActions content (it's always the same thing) ?

Reply:
As discussed, it could be replaced by a constant and check the `action` condition to store `null` when required.